tests/unit/backend/helpers/jest-db.ts

```diff
@@ -73,7 +87,25 @@ const insertFormCollectionReqs = async ({
     agency: agency.id,
   })
 
-  return { agency, user }
+  const encryptForm = await EncryptForm.create({
```

Comment:
Can make this into a separate function? If you foresee this being used only in one file, can even inline the creation of form as a function in that test file.

Prefer not to pollute `insertFormCollectionReqs`, as this is used by many tests and creation of forms/submissions may not be required for those tests (basically extend test times for no benefit)

Reply:
agreed. moved out